### ISSUE TITLE
v0.3.3 tarball's dist folder does not contain mjs

### ISSUE THREAD

**Issue by ngzhian** [acornjs/acorn-numeric-separator#3]
```
$ curl https://registry.npmjs.org/acorn-numeric-separator/-/acorn-numeric-separator-0.3.3.tgz | tar ztvf -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7736  100  7736    0     0   160k      0 --:--:-- --:--:-- --:--:--  160k
drwxr-xr-x 0/0               0 2020-06-13 08:17 package
-rw-r--r-- 0/0               6 2020-06-13 08:16 package/.eslintignore
-rw-r--r-- 0/0            9264 2020-06-13 08:16 package/.eslintrc.json
-rw-r--r-- 0/0             480 2020-06-13 08:16 package/CHANGELOG.md
-rw-r--r-- 0/0            1064 2018-09-14 02:17 package/LICENSE
-rw-r--r-- 0/0            1089 2020-06-13 08:16 package/README.md
drwxr-xr-x 0/0               0 2020-06-13 08:07 package/dist
-rw-r--r-- 0/0            1051 2020-06-13 08:16 package/package.json
-rw-r--r-- 0/0             332 2020-06-13 08:16 package/rollup.config.js
-rw-r--r-- 0/0             737 2020-05-03 05:19 package/run_test262.js
drwxr-xr-x 0/0               0 2020-06-13 08:16 package/src
drwxr-xr-x 0/0               0 2020-06-13 08:00 package/test
-rw-r--r-- 0/0            6187 2020-06-13 08:17 package/dist/acorn-numeric-separator.js
-rw-r--r-- 0/0            6034 2020-06-13 08:16 package/src/index.js
-rw-r--r-- 0/0              45 2017-12-16 15:17 package/test/.eslintrc.json
-rw-r--r-- 0/0            7600 2020-05-03 05:19 package/test/test.js
```

See how dist only contains acorn-numeric-separator.js, but not acorn-numeric-separator.mjs.

I tried running `npm pack --dry-run` locally:

```
$ npm pack --dry-run 2>&1 | grep mjs
src/index.js → dist/acorn-numeric-separator.js, dist/acorn-numeric-separator.mjs...
created dist/acorn-numeric-separator.js, dist/acorn-numeric-separator.mjs in 65ms
npm notice 13.4kB dist/acorn-numeric-separator.mjs.map
npm notice 6.2kB  dist/acorn-numeric-separator.mjs
```

and the mjs should be included in the tar.

**Comment by lukastaegert** [comment]
Bump as this is currently breaking our build.

**Comment by adrianheine** [comment]
Fixed with 0.3.5. Just creating a new pack didn't work for me with yarn, though.
